LayoutTests/fast/canvas/webgl/colorspaces-test.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>WebGL Canvas Conformance Tests</title>
+<script src="../../../resources/js-test.js"></script>
+<script src="resources/webgl-test.js"></script>
+<script src="resources/webgl-test-utils.js"></script>
+</head>
+<body>
+<div id="description"></div>
+<div id="console"></div>
+
+<script>
+description("This test ensures WebGL implementations correctly implement drawingbufferColorSpace.");
+
+function near(a, b, epsilon) {
+  return (a >= b - epsilon) && (a <= b + epsilon);
+}
+
+function checkPixels(gl, w, h, color, alpha) {
```

Comment:
I think similar function should exist in WebGLTestUtils

Reply:
checkCanvasRectColor may indeed do the job.